app/queue/queue.py

```diff
@@ -145,7 +145,7 @@ def handle_message(message, event_producer):
     with PayloadTrackerContext(payload_tracker, received_status_message="message received"):
         (output_host, add_results) = add_host(validated_operation_msg["data"])
         event = build_egress_topic_event(add_results.name, output_host, platform_metadata)
```

Comment:
```suggestion
        with metrics.egress_event_serialization_time.time():
            event = build_event(add_results.name, output_host, platform_metadata=platform_metadata)
```

And [analogically](https://github.com/RedHatInsights/insights-host-inventory/blob/aca1a0e48d0858746369dacdacffa186f7b9f572/api/host.py#L241):

```python
    event = build_event("updated", host, request_id=threadctx.request_id)
```